ibm/service/database/resource_ibm_database.go

```diff
@@ -2209,6 +2237,14 @@ func resourceIBMDatabaseInstanceUpdate(context context.Context, d *schema.Resour
 				change.Old = nil
 			}
 
+			newName, ok := change.New["name"]
+			isPgOrEdb := strings.Contains(service, "postgresql") || strings.Contains(service, "enterprisedb")
```

Comment:
PG only

Reply:
Logical Replication is PG only :D But the repl user exists in both edb and pg. So we need to handle it for both cases.